lib/Agrammon/Web/Routes.pm6

```diff
@@ -305,36 +305,38 @@ sub api-routes (Str $schema, $ws) {
         }
         operation 'exportExcel', -> LoggedIn $user {
             request-body -> %params {
+                # prevent header injection
+                my $filename = "%params<datasetName>.subst(/<-[\w\s_.-]>/, '', :g).xlsx";
```

Comment:
Maybe only allow spaces, not all whitespace, otherwise a newline could sneak in (I believe Cro will die if that happens, rather than let the header attack happen, but nice to prevent it here).

Reply:
Fixed.